src/components/workflow/workflow.service.ts

```diff
@@ -24,128 +45,120 @@ export class WorkflowService {
           [
             node('token', 'Token', {
               active: true,
-              value: session.token
+              value: session.token,
             }),
             relation('in', '', 'token', {
-              active: true
+              active: true,
             }),
             node('user', 'User'),
             relation('in', '', 'admin', {
-              active: true
+              active: true,
             }),
             node('baseNode', 'BaseNode', {
-              id: input.baseNodeId
+              id: input.baseNodeId,
             }),
           ],
         ])
         .merge([
           [
             node('baseNode'),
             relation('out', '', 'workflow', {
-              active: true
+              active: true,
             }),
             node('workflow', 'Workflow', {
               id: workflowId,
               stateIdentifier: input.stateIdentifier,
-            })
+            }),
           ],
         ])
         .merge([
           [
             node('workflow'),
             relation('out', '', 'possibleState', {
               active: true,
-              startingState: true
+              startingState: true,
             }),
             node('state', 'State', {
               id: generate(),
-              value: input.startingStateName
-            })
+              value: input.startingStateName,
+            }),
           ],
         ])
         .merge([
           [
             node('baseNode'),
             relation('out', '', `${input.stateIdentifier}`, {
-              active: true
+              active: true,
             }),
             node('currentState', 'CurrentState:Property', {
               active: true,
-              value: input.startingStateName
-            })
+              value: input.startingStateName,
+            }),
           ],
         ])
         .return({
-          workflow: [
-            { id: 'id' },
-            { stateIdentifier: 'stateIdentifier' },
-          ],
-          state: [
-            { id: 'startingStateId' },
-            { value: 'startingStateValue' }
-          ]
+          workflow: [{ id: 'id' }, { stateIdentifier: 'stateIdentifier' }],
+          state: [{ id: 'startingStateId' }, { value: 'startingStateValue' }],
         })
         .first();
 
-    if( !result ) {
-      throw new NotFoundException('could not create a workflow');
-    }
-
-    return {
-      id: result.id,
-      stateIdentifier: result.stateIdentifier,
-      startingState: {
-        id: result.startingStateId,
-        value: result.startingStateValue
+      if (!result) {
+        throw new NotFoundException('could not create a workflow');
```

Comment:
Couldn't create and not found are two different things.

Also this error is immediately replaced with the catch below